mkdocs.yml

```diff
@@ -79,6 +76,8 @@ markdown_extensions:
         - name: mermaid
           class: mermaid
           format: "!!python/name:pymdownx.superfences.fence_code_format"
+  - pymdownx.tabbed
+  - pymdownx.tasklist
```

Comment:
it seems `pymdownx.superfences` was removed in the commit `Add and sort mkdocs markdown extensions`. Was this deliberately done?

Reply:
Yes, it was duplicated. I clarified it in the commit.